src/point_one/fusion_engine/messages/data_version.h

```diff
@@ -75,6 +77,8 @@ std::ostream& operator<<(std::ostream& stream, const DataVersion& ver);
 
 std::string ToString(const DataVersion& ver);
 
+DataVersion FromString(const char* str);
```

Comment:
For convenience, might be nice to have an inline `std::string` overload that calls this one